### ISSUE TITLE
Add ARIA landmark to Pub edit area

### ISSUE THREAD

**Issue by gabestein** [pubpub/pubpub#546]
**What's your request?**
"The actual document entry field is buried within the page. Sighted users can simply see it and click; screen reader users do not have this option."

"At the very least, the phrase "begin writing here" should be marked up as a heading (perhaps h2). Landmarks are nice because they provide the ability to mark up large sections of the page with semantic labels which are only visible to those using screen readers. For instance, the main element provides a container which can be navigated to like a heading, but announces itself as "main" even though the text "main" is not on-screen. You can also provide additional label text via aria-label. Each page should have at least main, header, and footer elements. Most pages will also have a nav element since most pages will have elements used for navigation."

https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark

**What would this feature help you accomplish, and why is that important to you?**
- Allow screen reader users to quickly jump to editing.

**What's the URL to your PubPub community?**

### Optional

---

**Are there any other products or services that do this well?**

**Any attachments or reference images? (Drag them here)**

**Who requested this? (If not you)***
rjc / ATIC
